package/homebrew.spec

```diff
@@ -0,0 +1,90 @@
+%define homebrew_user linuxbrew
+%define homebrew_directory /home/%{homebrew_user}/.%{homebrew_user}
+%define brew_repo https://github.com/Homebrew/brew
+
+Summary:   Homebrew package manager
+Name:      homebrew
+Version:   0.0.0
+Release:   %autorelease
+License:   BSD-2-clause
+Group:     Development/Tools
+URL:       https://brew.sh
+Source:    %{brew_repo}/archive/refs/tags/%{git_version}.tar.gz#/brew.tar.gz
+
+# See: https://github.com/Homebrew/install/blob/master/install.sh#L211-L214
+BuildRequires: git
+
+Requires: procps-ng
+Requires: file
+Requires: gcc
+Requires: git >= 2.7.0
+Requires: glibc >= 2.13
+Requires: ruby >= 2.6.0
+
+%description
+The Missing Package Manager for macOS (or Linux)
+
+%global debug_package %{nil}
+%global _missing_build_ids_terminate_build 0
+%global __brp_mangle_shebangs /usr/bin/true
+
+%prep
+%autosetup -n brew-%{version}
+
+%build
+git init
+git remote set-url origin %{brew_repo} || git remote add origin %{brew_repo}
+
+%install
+install -d "$RPM_BUILD_ROOT%{homebrew_directory}"
+cp -r bin "$RPM_BUILD_ROOT%{homebrew_directory}"
```

Comment:
Yes and no, depends on what should be achieved. There are some people who choose to not use glob in order to catch if upstream changed some files that it would expect to have. In this case upstream will have full control so it is reasonable to use globs to copy everything relevant. Just keep in mind if there is anything that should not be published, e.g. for licensing issues or whatnot.

Reply:
yeh, I think it's better to use globs, thanks!

Reply:
Uhm, should glob all in the build directory (in this case the git repo archive) and move it to the linuxbrew user directory?

Reply:
Yes.